content/language/ruby/develop.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Use containers for Ruby on Rails development
+keywords: ruby, local, development
+description: Learn how to develop your Ruby on Rails application locally.
+---
+
+## Prerequisites
+
+Complete [Containerize a Ruby on Rails application](containerize.md).
+
+## Overview
+
+In this section, you'll learn how to set up a development environment for your containerized application. This includes:
+
+- Adding a local database and persisting data
+- Configuring Compose to automatically update your running Compose services as you edit and save your code
+
+## Add a local database and persist data
+
+You can use containers to set up local services, like a database. In this section, you'll update the `compose.yaml` file to define a database service and a volume to persist data.
+
+In the cloned repository's directory, open the `compose.yaml` file in an IDE or text editor. You need to add the database password file as an environment variable to the server service and specify the secret file to use.
+
+The following is the updated `compose.yaml` file.
+
+```yaml {hl_lines="09-30"}
```

Comment:
I think it's these lines. I might be off though.
```suggestion
```yaml {hl_lines="07-25"}
```

Reply:
done